astroquery/esa/xmm_newton/core.py

```diff
@@ -266,5 +267,80 @@ def get_columns(self, table_name, *, only_names=True, verbose=False):
         else:
             return columns
 
+    def get_epic_metadata(self, *, target_name=None,
+                          coordinates=None, radius=None):
+        """Downloads the European Photon Imaging Camera (EPIC)
+        metadata from a given target
+
+        Parameters
+        ----------
+        target_name : string, optional, default None
+            The name of the target
+        coordinates : astropy.coordinates.SkyCoord, optinal, default None
+            The coordinates of the target in a SkyCoord object
+        radius : float, optional, default None
+            The radius to query the target in degrees
+        Returns
+        -------
+        Four astropy.table.table.Table containing the metadata
+        of the given target
```

Comment:
Fixes to the docs rendering:

```suggestion
        coordinates : `~astropy.coordinates.SkyCoord`, optinal, default None
            The coordinates of the target in a SkyCoord object
        radius : float, optional, default None
            The radius to query the target in degrees

        Returns
        -------
        epic_source,  cat_4xmm, stack_4xmm, slew_source : `~astropy.table.Table` objects
            Tables containing the metadata of the target
        """
```